plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/OtelTelemetrySettings.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry;
+
+import io.opentelemetry.exporter.logging.LoggingSpanExporter;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import org.opensearch.SpecialPermission;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.telemetry.tracing.exporter.SpanExporterFactory;
+
+/**
+ * Otel specific telemetry settings.
+ */
+public class OtelTelemetrySettings {
+
+    /**
+     * Base Constructor.
+     */
+    public OtelTelemetrySettings() {}
```

Comment:
```suggestion
    private OtelTelemetrySettings() {}
```

Reply:
This class is (for now) just a holder, it is not supposed to be created, we could make it such